add limiter and gonify to third party (#46)
gofiber/awesome-fiber: README.md

List of middlewares that are created by the Fiber community.

- [shareed2k/fiber_tracing](https://github.com/shareed2k/fiber_tracing) - Middleware trace requests on Fiber framework with OpenTracing API.

> - [narmadaweb/limiter](https://github.com/narmadaweb/limiter) - Limiter using redis and three algorithm like fixed-window, sliding-window and token-bucket for fiber.

Reviewer: Why are you removing entries?

Author: okay i will change that, but i think the previous party's third was not in the update for too long

- [ansrivas/fiberprometheus](https://github.com/ansrivas/fiberprometheus) - Prometheus middleware for gofiber.
- [sacsand/gofiber-firebaseauth](https://github.com/sacsand/gofiber-firebaseauth) - Fiber Firebase Auth Middleware.
- [aschenmaker/fiber-health-check](https://github.com/aschenmaker/fiber-health-check) - Health-check middleware support health-check for Fiber️ framework.
- [elastic/apmfiber](https://github.com/elastic/apm-agent-go/tree/master/module/apmfiber) - APM Agent for Go Fiber.
- [eozer/fiber_ldapauth](https://github.com/eozer/fiber_ldapauth) - LDAP Authentication Middleware for Fiber.
- [darkweak/souin](https://github.com/darkweak/souin) - HTTP cache, RFC compliant, alternative to Varnish available as a middleware.
- [witer33/fiberpow](https://github.com/witer33/fiberpow) - Anti DDoS/Bot Middleware with a customizable Proof Of Work challenge.

> - [narmadaweb/gonify](https://github.com/narmadaweb/gonify) - Fiber Minifying middleware for HTML5, CSS3, JavaScript, Json, XML and SVG.

Reviewer: Same here, entry was removed

- [streamerd/fibergun](https://github.com/streamerd/fibergun) - A GunDB middleware for Fiber. Enables easy integration of GunDB, a decentralized database.
- [apitally/apitally-go](https://github.com/apitally/apitally-go) - Simple API monitoring tool for Fiber. Tracks API usage, errors, and performance, and includes request logging and alerting features.
- [newrelic/go-agent](https://github.com/newrelic/go-agent/tree/master/v3/integrations/nrfiber) - Official New Relic middleware for Fiber that manages instrumentation for New Relic monitoring.

> - [narmadaweb/limiter](https://github.com/narmadaweb/limiter) - Limiter using redis and three algorithm like fixed-window, sliding-window and token-bucket for fiber.

Reviewer 2: ```suggestion
- [narmadaweb/limiter](https://github.com/narmadaweb/limiter) -A high-performance Redis-backed rate limiter middleware for Fiber, supporting fixed window, sliding window, and token bucket algorithms.
```

> - [narmadaweb/limiter](https://github.com/narmadaweb/limiter) -A high-performance Redis-backed rate limiter middleware for Fiber, supporting fixed window, sliding window, and token bucket algorithms.

Reviewer 3: ```suggestion
- [narmadaweb/limiter](https://github.com/narmadaweb/limiter) - A high-performance Redis-backed rate limiter middleware for Fiber, supporting fixed window, sliding window, and token bucket algorithms.
```

> - [narmadaweb/gonify](https://github.com/narmadaweb/gonify) - Fiber Minifying middleware for HTML5, CSS3, JavaScript, Json, XML and SVG.

Reviewer 2: ```suggestion
- [narmadaweb/gonify](https://github.com/narmadaweb/gonify) - A Fiber middleware for minifying HTML5, CSS3, JavaScript, JSON, XML, and SVG content.
```
